src/test/java/com/opipo/codechallenge/config/TransactionToTransactionEntityMapIT.java

```diff
@@ -0,0 +1,81 @@
+package com.opipo.codechallenge.config;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.math.BigDecimal;
+import java.time.OffsetDateTime;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import com.opipo.codechallenge.CodechallengeApplication;
+import com.opipo.codechallenge.repository.model.TransactionEntity;
+import com.opipo.web.api.model.Transaction;
+
+@SpringBootTest(classes = CodechallengeApplication.class)
+@DisplayName("TransactionToTransactionEntityMap Integration test")
+public class TransactionToTransactionEntityMapIT {
+
+    @Autowired
+    private ModelMapper modelMapper;
+
+    @Test
+    @DisplayName("Given Transaction then transform it to TransactionEntity")
+    void givenTransactionThenTransformToTransactionEntity() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/app/albertinizao/codechallenge/pullRequest?prid=4681930)